src/screen/GameScreen.java

```diff
@@ -915,4 +927,14 @@ public void levelDamage(){
 			this.lives = 0;
 		}
 	}
+
+	/**
+	 * Updates the current ping value from the NetworkManager.
+	 */
+
+	private void updatePing() {
+		this.ping = NetworkManager.getLatency();
+	}
```

Comment:
I think is unnecessary method definition.
Just use `this.ping = NetworkManager.getLatency();`